packages/fdc3-context/schemas/context/order.schema.json

```diff
@@ -48,6 +53,7 @@
     {
       "type": "fdc3.order",
       "name": "...",
+      "notes": "...",
```

Comment:
I have a draft local PR in which I'm adding unit tests that verify the examples against the schema and this (and the one in product) will fail since they do not adhere to the schema.